### PR TITLE
Create Basic Front-end and Back-end Availability

### PR DESCRIPTION
**Pull request by betrevisan**
Availability will handle displaying a list of time slots a given tutor would be available to teach. I created the html file to display that (which uses bootstrap), a java servlet, and a file to write tests that refer to that servlet.

### CODE REVIEW

File: src/main/webapp/availability.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+
+        <div class="container-fluid">
+            <ul class="list-group" id="timeslots"></ul>
+        </div>
+        
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    </body>
+</html>
```
Comment (samfalberg) [inline src/main/webapp/availability.html:38]: New line

File: src/main/javatests/com/google/AvailabilityTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class SearchTest {
+
+
+}
```
Comment (samfalberg) [inline src/main/javatests/com/google/AvailabilityTest.java:26]: New line

Comment (betrevisan) [reply src/main/javatests/com/google/AvailabilityTest.java]: Thanks! I just updated those files that had new lines missing.

File: src/main/webapp/availability.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <meta charset="UTF-8">
+        <title>Availability</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    </head>
+    <body>
+        <div class="container-fluid">
+            <h1>Your Tutor's Availability</h1>
+            <h2>Select one of the free time slots below to shcedule your tutoring session.</h2>
```
Comment (kashiish) [inline src/main/webapp/availability.html:28]: schedule

Comment (betrevisan) [reply src/main/webapp/availability.html:28]: Thanks! I'll update that one.

**Pull request merged**